cluster/addons/fluentd-gcp/fluentd-gcp-configmap-old.yaml

```diff
@@ -0,0 +1,425 @@
+# This ConfigMap is used to ingest logs against old resources like
```

Comment:
NOTE: This file is roughly a copy of `fluentd-gcp-configmap.yaml`. The only difference is whether to set up `local_resource_id` and enable talking to Metadata Agent. With those, logs are ingested against `k8s_container` and `k8s_node`. Without those, logs are ingested against `gke_container` and `gce_instance`.

Reply:
+1 to what you've done here.